cirq-core/cirq/transformers/dynamical_decoupling.py

```diff
@@ -20,7 +20,6 @@
 
 from cirq.transformers import transformer_api
 from cirq.transformers.analytical_decompositions import single_qubit_decompositions
-from cirq.transformers.analytical_decompositions import unitary_to_pauli_string
 import cirq
```

Comment:
I just noticed you import `cirq`. this will cause circular imports.
```suggestion
from cirq.protocols import unitary_protocol
```